src/main/resources/entity-types/inventory/simple_instance.json5

```diff
@@ -151,7 +162,7 @@
         dataType: 'stringType',
       },
       isIdColumn: false,
-      queryable: true,
+      queryable: false,
```

Comment:
This change feels a little out of place? Should it be here?

Reply:
Oh yeah, I forgot to remove this. Thanks @mweaver-ebsco